plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConfig.java

```diff
@@ -70,6 +71,24 @@ public FakerConfig setLocale(String value)
         return this;
     }
 
+    @Max(2)
+    @Min(0)
+    public double getSequenceMinDistinctValuesRatio()
+    {
+        return sequenceMinDistinctValuesRatio;
+    }
+
+    @Config("faker.sequence-min-distinct-values-ratio")
+    @ConfigDescription(
+            """
+            Minimum ratio of distinct values of a column to total number of rows in a table to treat the columns as a sequence
```

Comment:
I didn't get the rationale for this, sounds like you want to treat almost unique value columns as sequences.

Reply:
Yes. I want to use sequences to make the values more unique. For example, when generating 1k values in the range of `[0, 1000)`, you won't get 1k unique values. Generating a sequence is easier to implement than a true unique random value generator.